apps/molgenis-viz/src/components/icons/viz-bar-chart.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
are these copied from some icon collection are constructed by out team ? is there some info in the readme on how to add to this icon set  ? or are these one offs that are specific to these components

Reply:
These are custom designed icons.